app/celery/process_delivery_status_result_tasks.py

```diff
@@ -68,7 +70,7 @@ def process_delivery_status(
         price_in_millicents_usd,
     ) = _get_notification_parameters(notification_platform_status)
 
-    # retrieves the inbound message for this provider we are updating the status of the outbound message
+    # Retrieve the inbound message for this provider.  (We are updating the status of the outbound message.)
```

Comment:
Nit: grammar. parenthetical statements go inside the sentence. it should be either
`# Retrieve the inbound message for this provider (we are updating the status of the outbound message).`

or 
`# Retrieve the inbound message for this provider. We are updating the status of the outbound message.`

Reply:
updated